docs/sources/reference/playlist.md

```diff
@@ -11,59 +11,96 @@ weight = 4
 
 # Playlist
 
-The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, or just show off your metrics to your team or visitors.
+The Playlist is a special type of Dashboard that rotates through a list of Dashboards. A Playlist can be a great way to build situational awareness, and show off your metrics to your team or visitors.
 
-Since Grafana automatically scales Dashboards to any resolution they're perfect for big screens!
+Grafana automatically scales Dashboards to any resolution, which makes them perfect for big screens.
+
+Access the Playlist feature from Grafana's side menu, in the Dashboards submenu.
 
 ## Create a playlist
 
 {{< docs-imagebox img="/img/docs/v50/playlist.png" max-width="25rem" class="docs-image--right">}}
 
-The Playlist feature can be accessed from Grafana's sidemenu, in the Dashboard submenu.
+To create a playlist: 
+
+1. Click the __New playlist__ button.
+1. Name your playlist in the __Name__ text box.
+1. Enter a time interval in the __Interval__ text box.
```

Comment:
What are my options here? It looks like the default is 5m, which I assume is 5 minutes. Can I only do minutes? What is the format for seconds or hours? Can I mix them, like `1m 30s`?

What if I enter multiple intervals? Can I display the first dashboard for 1m, the second for 2m, and so on, by entering `1m,2m,3m,4m`?

Reply:
I'm saving this question for @marcusolsson ! Will post in the docs channel.